MVVM-C Scene.xctemplate/___VARIABLE_sceneIdentifier___.storyboard

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="12120" systemVersion="16E195" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="hvM-iD-sv8">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="12121" systemVersion="16G29" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="hvM-iD-sv8">
     <device id="retina4_7" orientation="portrait">
         <adaptation id="fullscreen"/>
     </device>
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="12088"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="12089"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
-        <!--View Controller-->
+        <!--scene Identifier View Controller-->
         <scene sceneID="wNu-vz-AgM">
             <objects>
-                <viewController storyboardIdentifier="___VARIABLE_sceneIdentifier___ViewControllerID" useStoryboardIdentifierAsRestorationIdentifier="YES" id="mrg-aK-cPW" customClass="___VARIABLE_sceneIdentifier___ViewController" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController storyboardIdentifier="___VARIABLE_sceneIdentifier___ViewControllerID" automaticallyAdjustsScrollViewInsets="NO" useStoryboardIdentifierAsRestorationIdentifier="YES" id="mrg-aK-cPW" customClass="___VARIABLE_sceneIdentifier___ViewController" customModuleProvider="target" sceneMemberID="viewController">
```

Comment:
Jaký je důvod tohoto nastavení?

Reply:
vacsinou tu fuckuje scrollViews vo viewControlleroch, napr na CarPics aj Pikolikovi som to niekde musel vypinat a zistil som ze je to super vypnut vzdy :D (len tusim na iOS11 ti to robi efekt pri push a pop ked to mas zapnute, otazne je ci sa vam ten efekt paci a ci ho chceme pouzivat)

Reply:
Právě, že na iOS 11 bych to i kvůli iPhone X nevypínal. Kupříkladu já to například nikdy nevypínám, takže je to dost osobní preference. A od toho jsou standardní nastavení.

Reply:
Souhlasím s @mkj-is, nechme to prosím na výchozí hodnotě (YES) ať se to chová stejně jako když přidáváš controller do storybordu ručně. Jeden z argumentů je právě ten iPhone X, další je např. aby to nemátlo ostatní, kteří jsou zvyklí na výchozí chování.

Reply:
okk

Reply:
Revertnul bych ten řádek, aby nebyla změna, ale to je detail. ;)